examples/rsp-cra-18/src/sections/CollectionExamples.tsx

```diff
@@ -0,0 +1,83 @@
+import {ActionMenu, Flex, Divider, Item, ListBox, ListView, MenuTrigger, Menu, ActionButton, TableBody, TableView, Row, Cell, TableHeader, Column, TagGroup} from '@adobe/react-spectrum';
+
+export default function CollectionExamples(){
```

Comment:
Should any collection examples have dynamic or API data?

Reply:
i think this would be a good idea. probably would be a follow-up